cookbook/slackbot/start.py

```diff
@@ -154,20 +154,37 @@ async def chat_endpoint(request: Request):
         raise HTTPException(400, "Invalid event type")
     match payload.type:
         case "event_callback":
-            channel_name = await get_channel_name(payload.event.channel)
-            if channel_name.startswith("D"):
-                # This is a DM channel, we should not respond
-                logger.warning(f"Attempted to respond in DM channel: {channel_name}")
-                slack_webhook = await SlackWebhook.load("marvin-bot-pager")
-                await slack_webhook.notify(
-                    body=f"Attempted to respond in DM channel: {channel_name}",
-                    subject="Slackbot DM Warning",
+            print(payload.event)
```

Comment:
let's change this to an assert so that our type checker knows for sure that `payload.event` is not equal to `None`
```suggestion
            assert payload.event is not None, "No event found!"
```